src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+4. The [Figma diagram](https://www.figma.com/file/8LeT0Eq7a3ylAyM19c0FJp/ASC%3A-Tooltip?node-id=110%3A0) (as of February 2021) [FD]
+5. Email thread (MS internal) [ET] 
+6. WPF [Tooltip Overview](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/tooltip-overview?view=netframeworkdesktop-4.8) (and related links) [TO]
+
+The first three describe behavior in general terms, guiding any UI platform (WCAG) or Microsoft UI platforms (MAS).
+The Figma diagram has more specific requirements, but is not a complete spec.
+The email thread among many relevant MS teams helped fill in omissions, corrected errors, and clarified ambiguities in the diagram.
+The Tooltip Overview documents the existing behavior in WPF as of .NET 4.7.2.
+
+# Applicability
+
+All the changes described here are required to appear in .NET 6.0.
+
+The behavior changes will also appear in servicing updates for earlier versions as follows:
+- .NET 4.6-4.7.2, .NET 4.8:  the next biannual accessibility update (expected summer 2021)
+- .NET Core 3.1, .NET 5.0:  same time frame as above, patch version TBD
+
+The API changes (and behavior that depends on them) are only in .NET 6.0;
+servicing updates cannot include changes to public API.
+
+In the servicing updates, the new behavior is opt-in by setting
+`Switch.UseLegacyAccessibilityFeatures.5` to false, as described [here](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
+
+
+
+# Conceptual pages (How To)
+
+## Declaring a tooltip
+You can declare a tooltip in XAML or code as described in [TO].
+Starting in .NET 6.0, your declarations can also use the properties
+`ToolTip.ShowsToolTipOnKeyboardFocus`, and
+`ToolTipService.ShowsToolTipOnKeyboardFocus`.
+
+## Opening a tooltip
+When a parent element declares a tooltip, there are four ways to open (show) the tooltip:
+
+1. Hover the mouse over the parent.
+2. Move keyboard focus to the parent.
+3. Type a keyboard shortcut.
+4. Open the tooltip programmatically.
+
+Unless mentioned otherwise below, the tooltip is shown immediately and its size and position are governed by the placement properties described in [TO].
+
+### Hover
+A hover occurs when the mouse enters the parent element and stays there for a certain time, without any other activity.
+The tooltip is opened after the hover time, whose value is a function of the `InitialShowDelay` and `BetweenShowDelay` properties, and whether another tooltip is already showing [TO].
+
+### Keyboard focus
+When you use keyboard navigation to move focus to the parent element, the tooltip opens.
+(Using the mouse to move focus does not show the tooltip.)
+You can override this behavior using the `ShowsToolTipOnKeyboardFocus` properties.
+
+### Keyboard shortcut
+When the parent has keyboard focus, typing Ctrl+Shift+F10 opens the tooltip.
```

Comment:
What happens if the app or the system has registered this key combo? Does the notification still reaches the app or this will be filtered out by the WPF framework? Is the app or the framework handling it first?
Also I assume we want this key combo to work for any app, written in whatever framework and language, so this hot key need to be identical across Microsoft products. Let's make sure we mention this to CELA.

Reply:
CELA defined this key combo, based on the implementation already in Office apps. So Office suggested this combo, CELA approved the suggestion. This key combination is coming from the overall platform-agnostic guidance doc.

Reply:
It comes from [FD], the spec that's driving "identical across MS products".    CELA is probably the source.